portfolio/src/main/webapp/script.js

```diff
@@ -38,8 +32,114 @@ function addRandomOpinion() {
     opinionContainer.innerText = opinion;
 }
 
+function getdata() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+     const listmap = document.createElement("ul");   
+     comments.map(comment => {
+        const list = document.createElement("li");   
+        list.innerHTML = comment;
+
+         listmap.appendChild(list);
+     });
+
+    const commentsContainer = document.getElementById('comment-container');
+    commentsContainer.innerHTML = '';
+
+    commentsContainer.appendChild(listmap);
+        //createListElement(comments));
+
+
+
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
+
+
+  /*  const statsListElement = document.getElementById('hardcode-message-container');
+    statsListElement.innerHTML = '';
+    statsListElement.appendChild(
+        createListElement('Message 1: ' + messages[0]));
+    statsListElement.appendChild(
+        createListElement('Message 2 : ' + messages[1]));
+    statsListElement.appendChild(
+        createListElement('Message 3: ' + messages[2]));
+    console.log();
+    console.log(messages);
+    
+    */
+    
+  });
+}
+
+
+/** Creates an <li> element containing text. */
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
+}
+
 async function getDataUsingAsyncAwait() {
   const response = await fetch('/data');
   const data = await response.text();
   document.getElementById('data-container').innerText = data;
 }
+
+function createMap() {
+    const map = new google.maps.Map(
+        document.getElementById('map'),
+        {center: {lat: 37.6376938, lng: -122.0843484}, 
+        zoom: 9,
+        mapTypeId: google.maps.MapTypeId.HYBRID
+        });
+        
+    const Marker1 = new google.maps.Marker({
```

Comment:
The google JavaScript style guide recommends using lowerCamelCase for variable names: https://google.github.io/styleguide/jsguide.html#naming-local-variable-names

This might sound nitpicky, but  the motivation is to make it more obvious to somebody reading your code. When I see something that starts with an upper-case letter I assume it's a class. Funny enough so does GitHub, according to its syntax highlighting! 🙃

And actually now that I'm looking at it, I don't think you need these variables at all, since you don't reference them? I think the `new google.maps.Marker` call adds the markers to the map without you needing to keep a reference to the variables around?